common/templates/discussion/index.html

```diff
@@ -28,4 +30,25 @@
 
 <%include file="_discussion_course_navigation.html" args="active_page='discussion'" />
 
-<%include file="/discussion/_discussion_course.html" />
+<section class="discussion container" id="discussion-container"
+         data-roles="${roles}"
+         data-course-id="${course_id | h}"
+         data-user-info="${user_info}"
+         data-threads="${threads}"
+         data-thread-pages="${thread_pages}"
+         data-content-info="${annotated_content_info}"
+         data-sort-preference="${sort_preference}"
+         data-flag-moderator="${flag_moderator}"
+         data-user-cohort-id="${user_cohort}"
+         data-course-settings="${course_settings}">
+    <div class="discussion-body">
+        <div class="forum-nav"></div>
+        <div class="discussion-column">
+          <article class="new-post-article" style="display: none"></article>
+          <div class="forum-content"></div>
+        </div>
+    </div>
+</section>
```

Comment:
@mtyaka no, it was like that in previous (pre-birch) version. I've extracted common parts in dedicated template, thank you for the suggestion.